docs/reference/release-notes/highlights.asciidoc

```diff
@@ -81,4 +81,23 @@ Composite aggregations on `keyword` fields no longer use
 {ref}/eager-global-ordinals.html[global ordinals], which for high cardinality
 fields could use a lot of heap memory as part of the
 {ref}/modules-fielddata.html[field data cache].
+
+[discrete]
+[[new-migrate-data-tiers-routing-api]]
+=== New migrate to data tiers routing API
+
+7.14 introduces the {ref}/ilm-migrate-to-data-tiers.html[migrate to data tiers
+routing API]. You can use the API to switch indices and ILM policies that use
+attribute-based allocation filters to data tiers. This lets ILM automatically
```

Comment:
Perhaps something (might be useful to link to the node roles docs for it too) like:
```suggestion
attribute-based allocation filters to data tiers using node roles. This lets ILM automatically
```

Reply:
Added!